### PR TITLE
cloud: update support doc for support plans

### PR DESCRIPTION
**Pull request by TomShawn**
### What is changed, added or deleted? (Required)
cloud: update support doc for support plans

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [ ] master (the latest development version)
- [ ] v6.2 (TiDB 6.2 versions)
- [x] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: tidb-cloud/tidb-cloud-support.md [hunk 1]
```diff
@@ -5,17 +5,65 @@ summary: Learn how to contact the support team of TiDB Cloud.
...
-3. Click **Send** to submit your ticket. After the support team receives your ticket, they will reply to you via email.
+To request support for your organization or project, perform the following steps:
 
-4. Check your email to view the reply, your ticket history, and communication records.
+1. Click **Support** on the upper right of the top navigation bar, and click **Create New Case**. A **Submit a request** page is displayed.
+
+    > **Note:**
+    >
+    > You can also directly click the **Help** icon on the lower right.
+å
```
Comment (TomShawn) [inline tidb-cloud/tidb-cloud-support.md:21]:
```suggestion

```

File: tidb-cloud/tidb-cloud-support.md [hunk 2]
```diff
@@ -5,17 +5,83 @@ summary: Learn how to contact the support team of TiDB Cloud.
...
+TiDB Cloud offers support services to your organization. You can use the free **Basic** support plan in TiDB Cloud or upgrade to a paid plan for extended services.
 
-1. Click **Support** on the upper right of the top navigation bar, and click **Request Support**. Or directly click the **Help** icon on the lower right.
+The information of each support plan is available on the support page of the TiDB Cloud console. To access the page, see [Check or upgrade your support plan](#check-or-upgrade-your-support-plan).
 
-2. In the **Leave us a message** dialog, fill in the ticket.
+## Request support
 
-3. Click **Send** to submit your ticket. After the support team receives your ticket, they will reply to you via email.
+To request support for your organization or project, perform the following steps:
```
Comment (qiancai) [inline tidb-cloud/tidb-cloud-support.md:14]:
```suggestion
To request support for your organization, project, or cluster, perform the following steps:
```

File: tidb-cloud/tidb-cloud-support.md [hunk 3]
```diff
@@ -5,17 +5,83 @@ summary: Learn how to contact the support team of TiDB Cloud.
...
 
-4. Check your email to view the reply, your ticket history, and communication records.
+1. Click **Support** on the upper right of the top navigation bar, and click **Create New Case**. A **Submit a request** page is displayed.
+
+    > **Note:**
```
Comment (qiancai) [inline tidb-cloud/tidb-cloud-support.md:18]:
```suggestion
    > **Tip:**
```

File: tidb-cloud/tidb-cloud-support.md [hunk 4]
```diff
@@ -5,17 +5,83 @@ summary: Learn how to contact the support team of TiDB Cloud.
...
+    >
+    > You can also directly click the **Help** icon on the lower right.
```
Comment (qiancai) [inline tidb-cloud/tidb-cloud-support.md:20]:
```suggestion
    > You can also directly click the **Help** icon on the lower right of the TiDB Cloud console.
```

File: tidb-cloud/tidb-cloud-support.md [hunk 5]
```diff
@@ -5,17 +5,83 @@ summary: Learn how to contact the support team of TiDB Cloud.
...
+
+2. On the **Submit a request** page, select **TiDB Cloud Support** from the dropdown list.
```
Comment (qiancai) [inline tidb-cloud/tidb-cloud-support.md:22]:
```suggestion
2. On the **Submit a request** page, select **TiDB Cloud Support** from the drop-down list.
```

File: tidb-cloud/tidb-cloud-support.md [hunk 6]
```diff
@@ -5,17 +5,83 @@ summary: Learn how to contact the support team of TiDB Cloud.
...
+
+3. Fill in the boxes below.
```
Comment (qiancai) [inline tidb-cloud/tidb-cloud-support.md:24]:
```suggestion
3. Fill in the request information.
```

File: tidb-cloud/tidb-cloud-support.md [hunk 7]
```diff
@@ -5,17 +5,83 @@ summary: Learn how to contact the support team of TiDB Cloud.
...
+
+4. Click **Submit** to submit your ticket. After the support team receives your ticket, they will reply to you via email.
+
+5. Check your email to view the reply, your ticket history, and communication records.
```
Comment (qiancai) [inline tidb-cloud/tidb-cloud-support.md:28]: Do we have ticket history in email? @zhoubasten

File: tidb-cloud/tidb-cloud-support.md [hunk 8]
```diff
@@ -5,17 +5,83 @@ summary: Learn how to contact the support team of TiDB Cloud.
...
+
+    > **Note:**
+    >
+    > You can also directly click the **Help** icon on the lower right.
+
+2. On the **Submit a request** page, select **TiDB Cloud Support** from the dropdown list.
+
+3. Fill in the boxes below.
+
+4. Click **Submit** to submit your ticket. After the support team receives your ticket, they will reply to you via email.
```
Comment (qiancai) [inline tidb-cloud/tidb-cloud-support.md:26]: Do we need to mention something like "The response time depends on the support plan of your organization? "

File: tidb-cloud/tidb-cloud-support.md [hunk 9]
```diff
@@ -5,17 +5,83 @@ summary: Learn how to contact the support team of TiDB Cloud.
...
+The information of each support plan is available on the support page of the TiDB Cloud console. To access the page, see [Check or upgrade your support plan](#check-or-upgrade-your-support-plan).
 
-2. In the **Leave us a message** dialog, fill in the ticket.
+## Request support
 
-3. Click **Send** to submit your ticket. After the support team receives your ticket, they will reply to you via email.
+To request support for your organization or project, perform the following steps:
 
-4. Check your email to view the reply, your ticket history, and communication records.
+1. Click **Support** on the upper right of the top navigation bar, and click **Create New Case**. A **Submit a request** page is displayed.
```
Comment (qiancai) [inline tidb-cloud/tidb-cloud-support.md:16]:
```suggestion
1. On the TiDB Cloud console, click **Support** in the upper-right corner of the top navigation bar, and click **Create New Case**. A **Submit a request** page is displayed.
```

Comment (qiancai) [reply tidb-cloud/tidb-cloud-support.md:16]: This comment is to align with line 39.

File: tidb-cloud/tidb-cloud-support.md [hunk 10]
```diff
@@ -5,17 +5,83 @@ summary: Learn how to contact the support team of TiDB Cloud.
...
+1. On your TiDB Cloud console, click **Support** in the upper-right corner of the top navigation bar, and click **View Support Plan**. The support page is displayed.
+2. In the **Request Support** area, click **View Support Portal**. The **My request** page is displayed.
+
+    On the **My request** page, you can view the status of your request.
+
+Alternatively, you can also access the [Customer Support](https://support.pingcap.com/hc/en-us) page, log in to the support portal, and then click **View my requests**.
+
+## Check or upgrade your support plan
+
+To check and upgrade your support plan, perform the following steps:
```
Comment (qiancai) [inline tidb-cloud/tidb-cloud-support.md:48]:
```suggestion
To check or upgrade your support plan, perform the following steps:
```

File: tidb-cloud/tidb-cloud-support.md [hunk 11]
```diff
@@ -5,17 +5,83 @@ summary: Learn how to contact the support team of TiDB Cloud.
...
+
+    To upgrade to **Standard**:
+
+    1. Click **Select Plan** in the **Standard** pane. A **Finish and Start Using Support** page is displayed.
+    2. Check the billing information in the lower-left corner of the page.
+    3. Add your payment information in the **Billing Profile** and **Credit Card** areas.
+
+        For more information about billing, see [TiDB Cloud Payment Method](/tidb-cloud/tidb-cloud-billing.md#payment-method).
+
+    4. Click **Confirm and Pay** in the lower-right corner of the page.
```
Comment (qiancai) [inline tidb-cloud/tidb-cloud-support.md:67]:
```suggestion
    4. Click **Confirm and Start Using Support** in the lower-right corner of the page.
```

File: tidb-cloud/tidb-cloud-support.md [hunk 12]
```diff
@@ -5,17 +5,83 @@ summary: Learn how to contact the support team of TiDB Cloud.
...
+
+    After the payment is finished, you have upgraded you plan to **Standard**.
```
Comment (qiancai) [inline tidb-cloud/tidb-cloud-support.md:69]: 9 月 30 号之前是否需要 payment 呢

File: tidb-cloud/tidb-cloud-support.md [hunk 13]
```diff
@@ -5,17 +5,83 @@ summary: Learn how to contact the support team of TiDB Cloud.
...
+
+    </div>
+    <div label="Upgrade to Enterprise or Premium">
+
+    To upgrade your plan to **Enterprise** or **Premium**:
+
+    1. Click **Contact Sales** in the **Enterprise** or **Premium** pane. A **Contact Us** page is displayed.
+    2. Fill in your information in the **Get in Touch** area and click **SUBMIT**. Then, the support team will contact you and help your further your subscription.
```
Comment (qiancai) [inline tidb-cloud/tidb-cloud-support.md:77]:
```suggestion
    2. Fill in and submit your contact information on the page. Then, the support team will contact you and help your further your subscription.
```

File: tidb-cloud/tidb-cloud-support.md [hunk 14]
```diff
@@ -5,17 +5,83 @@ summary: Learn how to contact the support team of TiDB Cloud.
...
+
+    </div>
+    </SimpleTab>
+
+## Downgrade your support plan
+
+To downgrade your support plan to **Basic**, perform the following steps:
+
+1. On your TiDB Cloud console, click **Support** in the upper-right corner of the top navigation bar, and click **View Support Plan**.
+2. At the lower part of the **Support** page, click **Downgrade to basic plan**.
```
Comment (qiancai) [inline tidb-cloud/tidb-cloud-support.md:87]:
```suggestion
2. At the bottom of the **Support** page, click **Downgrade to basic plan**.
```

File: tidb-cloud/tidb-cloud-support.md (hunk 8 above)
Comment (TomShawn) [reply tidb-cloud/tidb-cloud-support.md:26]:
```suggestion
4. Click **Submit** to submit your ticket. After the support team receives your ticket, they will reply to you via email. The response time depends on the support plan of your organization.
```

File: tidb-cloud/tidb-cloud-support.md (hunk 10 above)
Comment (TomShawn) [reply tidb-cloud/tidb-cloud-support.md:48]: 其实这里是连续的。。。

File: tidb-cloud/tidb-cloud-support.md (hunk 12 above)
Comment (TomShawn) [reply tidb-cloud/tidb-cloud-support.md:69]: @zhoubasten cc
收费是常态的话，也许文档里不用写免费信息？

**Review by zhoubasten — APPROVED** [review]
LGTM

File: tidb-cloud/tidb-cloud-support.md (hunk 12 above)
Comment (TomShawn) [reply tidb-cloud/tidb-cloud-support.md:69]: @zhoubasten 认为不写也行

File: tidb-cloud/tidb-cloud-support.md [hunk 15]
```diff
@@ -5,17 +5,76 @@ summary: Learn how to contact the support team of TiDB Cloud.
 
 # TiDB Cloud Support
 
-TiDB Cloud offers support services to your organization. To contact our support team, perform the following steps:
+TiDB Cloud offers support services to your organization. You can use the free **Basic** support plan in TiDB Cloud or upgrade to a paid plan for extended services.
```
Comment (qiancai) [inline tidb-cloud/tidb-cloud-support.md:8]:
```suggestion
TiDB Cloud offers a free **Basic** support plan in TiDB Cloud for each user and you can upgrade to a paid plan for extended services.
```

File: tidb-cloud/tidb-cloud-support.md [hunk 16]
```diff
@@ -5,17 +5,76 @@ summary: Learn how to contact the support team of TiDB Cloud.
...
+    >
+    > You can also directly click the **Help** icon on the lower right of the TiDB Cloud console.
 
-    - Replying to the email sent by the support team directly
-    - Clicking the ticket link attached in the email to go to the support portal to check and add comments to your ticket
+2. On the **Submit a request** page, select **TiDB Cloud Support** from the drop-down list.
+
+3. Fill in the request information.
+
+4. Click **Submit** to submit your ticket. After the support team receives your ticket, they will reply to you via email. The response time depends on the support plan of your organization.
```
Comment (qiancai) [inline tidb-cloud/tidb-cloud-support.md:26]:
```suggestion
4. Click **Submit** to submit your ticket. After the support team receives your ticket, they will reply to you via email. The response time depends on the [support plan](#check-or-upgrade-your-support-plan) of your organization.
```

File: tidb-cloud/tidb-cloud-support.md [hunk 17]
```diff
@@ -5,17 +5,76 @@ summary: Learn how to contact the support team of TiDB Cloud.
...
+
+    After the payment is finished, you have upgraded you plan to **Standard**.
+
+    </div>
+    <div label="Upgrade to Enterprise or Premium">
+
+    To upgrade your plan to **Enterprise** or **Premium**:
+
+    1. Click **Contact Sales** in the **Enterprise** or **Premium** pane. A **Contact Us** page is displayed.
+    2. Fill in and submit your contact information on the page. Then, the support team will contact you and help your further your subscription.
```
Comment (qiancai) [inline tidb-cloud/tidb-cloud-support.md:70]:
```suggestion
    2. Fill in and submit your contact information on the page. Then, the support team will contact you and help you with your subscription.
```

**Review by qiancai — APPROVED** [review]

**Pull request merged**